pkg/skaffold/debug/cnb.go

```diff
@@ -25,9 +25,9 @@ import (
 	cnbl "github.com/buildpacks/lifecycle/launch"
 	shell "github.com/kballard/go-shellquote"
 	"github.com/sirupsen/logrus"
-	v1 "k8s.io/api/core/v1"
 
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/debug/annotations"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/debug/types"
```

Comment:
they need to be in a subpackage because they're imported by multiple packages which would cause an import collision, but they can be consolidated into `pkg/skaffold/debug/types`